docs/en/00_Getting_Started/02_Composer.md

```diff
@@ -140,7 +140,7 @@ This package will maintain your [.gitignore](http://git-scm.com/docs/gitignore)
 
 Include the package in your project by running this command
 
-    composer require gdmedia/ss-auto-git-ignore
+    composer require gdmedia/ss-auto-git-ignore --dev
```

Comment:
I think this is not for require. how would you add a prod dependency then?
for update and install its the other way around, there dev ist the default and you need to use --no-dev to exclude dev deps.